litert/vendors/qualcomm/core/utils/miscs.cc

```diff
@@ -60,4 +72,85 @@ bool CreateDirectoryRecursive(const std::filesystem::path& dir_name) {
   return true;
 }
 
+DlHandle::DlHandle(const char* path)
+    : handle_(dlopen(path, RTLD_LAZY | RTLD_LOCAL)) {}
+
+DlHandle::~DlHandle() {
+  if (handle_) dlclose(handle_);
+}
+
+void* DlHandle::get() const { return handle_; }
+
+bool DlHandle::valid() const { return handle_ != nullptr; }
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Names
```suggestion
void* DlHandle::Get() const { return handle_; }

bool DlHandle::Valid() const { return handle_ != nullptr; }
```